src/migration/1725988464760-SourceMembersIndex.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class SourceMembersIndex1725988464760 implements MigrationInterface {
+  name = 'SourceMembersIndex1725988464760';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE INDEX "IDX_source_flags_total_members" ON post USING HASH (((flags->'totalMembers')::integer))`,
```

Comment:
Checked index is used with `EXPLAIN ANALYZE`?

Reply:
Haven't tried that, let me check.